datadog_checks_dev/datadog_checks/dev/tooling/commands/release/show/changes.py

```diff
@@ -23,6 +23,11 @@ def changes(ctx, check, tag_pattern, tag_prefix, dry_run):
 
     # get the name of the current release tag
     cur_version = get_version_string(check, pattern=tag_pattern, tag_prefix=tag_prefix)
+    if not cur_version:
+        abort(
+            'Failed to retrieve the latest version. Please ensure this has a proper set of tags following SemVer or '
+            'the provided tag_prefix/tag_pattern.'
+        )
```

Comment:
I agree the wording here is a bit clunky, I'm not sure `repository` is right since this can be a check_name as well that we can't find a tag for. I'm not sure what a better name would be though haha.